experiments/browser_based_querying/src/hackernews/adapter.ts

```diff
@@ -128,7 +129,7 @@ function* linksInAboutPage(
     }
 
     const aboutPlain = extractPlainTextFromHnMarkup(aboutHtml);
-    const matches2 = aboutPlain.matchAll(/http[s]?:\/\/[^ \n\t]*[^ \n\t\.);,\]}]/g);
```

Comment:
Are you sure about this change?

Reply:
the difference is we don't escape a . in a [] which is fine (and is a prettier error). I can revert if you want. https://eslint.org/docs/latest/rules/no-useless-escape

Reply:
Hmm interesting, I thought `[]` accepted character groups like `.` 🤔 

Any idea why our linters didn't already warn on this? We have a JS/TS lint job running in CI.

(Side note: as a norm in open-source development, it's usually better to fix things like these in a separate PR. It's easier to merge and offers the chance to explain why the change is being made. When you "slip it in" into some other PR, it risks looking accidental or even sloppy.)

Reply:
Understandable